commit 5183255f288e7f6cad4e15b93f14eb3f27cd0193 (refs/changes/64/2164/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2017-06-28T15:52:59-07:00 (2 years, 3 months ago)
    
    IMGAPI-632 URL encoded query string induces illegal instruction in jsSprintf

diff --git a/CHANGES.md b/CHANGES.md
index a0260b4..a6d9e38 100644
--- a/CHANGES.md
+++ b/CHANGES.md
@@ -1,5 +1,9 @@
 # IMGAPI changelog
 
+## 3.3.1
+
+- IMGAPI-632 Fix potential crash from in static served URLs.
+
 ## 3.3.0
 
 - IMGAPI-627 With this change, a standalone IMGAPI will use an ECDSA key for
diff --git a/package.json b/package.json
index f5bd79f..c6e7847 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "imgapi",
   "description": "Image API to manage images for SDC 7",
-  "version": "3.3.0",
+  "version": "3.3.1",
   "author": "Joyent (joyent.com)",
   "private": true,
   "dependencies": {
@@ -32,7 +32,7 @@
     "passwd": "0.0.11",
     "posix-getopt": "1.0.0",
     "progbar": "git+https://github.com/trentm/node-progbar.git#a4c56e6",
-    "restify": "4.3.0",
+    "restify": "4.3.1",
     "rimraf": "2.2.6",
     "sdc-clients": "10.0.4",
     "semver": "3.0.1",
